openshift/env-template.sh

```diff
@@ -48,7 +48,7 @@ export GITHUB_OAUTH_CONSUMER_SECRET='not-set'
 # Non-essential configuration options follow. You likely don't need to touch these.
 
 # Deployment prefix
-export DEPLOYMENT_PREFIX=${DEPLOYMENT_PREFIX:-${OC_USERNAME}}
+export DEPLOYMENT_PREFIX=${OC_USERNAME}
```

Comment:
@miteshvp With `OC_TOKEN` in place, `OC_USERNAME` and `OC_PASSWORD` can be deprecated.  I think we should set value to `DEPLOYMENT_PREFIX` env variable directly instead of relying on the value of `OC_USERNAME`. WDYT?

Reply:
With the recent changes, your aws resources should have deployment_prefix starting with your kerberos id
Maybe `export DEPLOYMENT_PREFIX=${OC_USERNAME}-dev`. WDYT?

Reply:
@samuzzal-choudhury ^^

Reply:
@miteshvp I meant to use only the `DEPLOYMENT_PREFIX` env variable to set a value. I do not think we need OC_USERNAME/_PASSWORD any more. They may cause confusion if kept in the env-template. But, we can live with that :-) PR looks good to me otherwise, hence, approving.